freqtrade/arguments.py

```diff
@@ -0,0 +1,247 @@
+"""
+This module contains the argument manager class
+"""
+
+import argparse
+import os
+import re
+import logging
+from typing import List
+
+from freqtrade import __version__
+from freqtrade.constants import Constants
+
+
+class Arguments(object):
```

Comment:
I think it's a good idea to refactor the argument handling out in an own module, but I think a class is overkill here.